Formula/texlab.rb

```diff
@@ -30,7 +30,7 @@ def install
         {"jsonrpc": "2.0", "id": 0, "method": "initialize", "params": { "rootUri": null, "capabilities": {}}}
 
       EOF
-      assert_match "Content-Length: 543", stdout.gets("\n")
+      assert_match "Content-Length:"
```

Comment:
You're not currently specifying data to match against
```suggestion
      assert_match "Content-Length:", stdout.gets("\n")
```

Reply:
D'oh!